src/components/testCaseLanding/qdm/TestCaseList.tsx

```diff
@@ -211,8 +210,45 @@ const TestCaseList = (props: TestCaseListProps) => {
       });
       setTestCases([...testCases]);
     }
+    clauseCoverageProcessor(calculationOutput);
   }, [calculationOutput, selectedPopCriteria]);
 
+  const clauseCoverageProcessor = (calculationOutput) => {
+    //generates current populations coverage %
+    if (calculationOutput && selectedPopCriteria) {
+      const trueSet = new Set<string>();
+      const allSet = new Set<string>();
```

Comment:
Style suggestion:
Perhaps `trueClauses` and `allClauses`. `Set` is kinda redundant since that's the type, but many developers like to put the type into the variable name to reference later.